sno/working_copy/postgis.py

```diff
@@ -287,6 +282,12 @@ def _create_table_for_dataset(self, sess, dataset):
             f"""CREATE TABLE IF NOT EXISTS {self.table_identifier(dataset)} ({table_spec});"""
         )
 
+    def _insert_or_replace_into_dataset(self, dataset):
```

Comment:
this could do with a link to the relevant docs.

Reply:
Done